### PR TITLE
Arm improvement and tuning

### LINKED ISSUE #52: Arm control improvements and tuning
**Summary**
The arm control needs to be improved. Specifically, the following things need to be added:

1. A feedforward term needs to be added to the arm rotate PID. Currently the motor power is decided purely with PID. This change would be to have `motor power = pid + ff` where ff is a feedforward variable. A good initial guess at what ff is could be:
```
ff = sign(setpoint) * Kf * setpoint^2
```
where `Kf` is some parameter file gain (probably something like 0.05 or 0.1), `sign` is there to ensure the feed forward goes in the direction the arm is going, and `setpoint` is the target value for the arm's rotation

2. Once feedforward is added, the arm rotate PID will need to be tuned. It is likely that the Kp will need to decrease, but it's worth trying to increase the Kd first to see how effective that is.
3. We should get the mech team to make the extend faster. This will likely require tuning the extend PID to not oscillate
4. Finally, we should reduce the open loop ramp rate limits on the arm so it can go faster. We need to do this carefully to make sure stuff doesn't break though.

**Work to do**
- Items 1-4 above, or a justification of why it doesn't work (or isn't effective)

**Verification**
- Robot testing

### PR DESCRIPTION
**Pull request by deadlyvortex**
Added feedforward to arm control.
Fixes #52

### CODE REVIEW

File: src/main/cpp/subsystems/Arm.cpp [hunk 1]
```diff
@@ -147,7 +152,8 @@ bool Arm::TestControl() {
     this->inches_pid_->SetSetpoint(this->desired_state_.extension);
 
     // set output
-    this->interface_->arm_lift_power = this->arm_pid_->Calculate(this->state_.rotation);
+    double ff = TeamOKC::sign(this->desired_state_.rotation) * arm_kF_ * this->desired_state_.rotation * this->desired_state_.rotation;
```
Comment (danielbrownmsm) [inline src/main/cpp/subsystems/Arm.cpp:155]: add a comment about what this is actually doing

File: src/main/deploy/parameters.toml [hunk 2]
```diff
@@ -88,6 +88,7 @@ arm_setpoint = 94.0
 intake_setpoint = -40.0
 
 [arm.negative_pickup]
+extend_setpoint = 20.0
 extend_setpoint = 50.0
```
Comment (danielbrownmsm) [inline src/main/deploy/parameters.toml:92]: why is there another extend setpoint here?

**Review by danielbrownmsm — CHANGES_REQUESTED** [review]
I'll go ahead and make these changes, but I'm leaving this review so Sai can see how the process works

**Comment by deadlyvortex** [comment]
ok

**Review by danielbrownmsm — APPROVED** [review]
lgtm, and then we can merge the feat/GKC branch

**Pull request merged**